app/src/main/res/layout/bottom_left_overlay.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="listener"
+      type="org.oppia.android.app.spotlight.SpotlightNavigationListener" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/overlayConstraintLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/transparent_background"
+    android:clickable="true"
+    android:focusable="true">
+
+    <ImageView
+      android:id="@+id/arrow"
+      android:layout_width="@dimen/arrow_width"
+      android:layout_height="@dimen/arrow_height"
+      android:rotationX="180"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      app:srcCompat="@drawable/ic_rounded_arrow_up_right" />
+
+    <TextView
+      android:id="@+id/custom_text"
+      style="@style/TextViewStart"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="8dp"
+      android:layout_marginEnd="10dp"
+      android:background="@drawable/spotlight_overlay_hint_background"
+      android:paddingStart="8dp"
+      android:paddingTop="4dp"
+      android:paddingEnd="4dp"
+      android:paddingBottom="4dp"
+      android:text="this is a custom text"
+      android:textSize="20dp"
+      app:layout_constraintBottom_toTopOf="@id/arrow"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="@+id/arrow" />
+
+    <TextView
+      android:id="@+id/close_target"
+      style="@style/StateButtonActive"
+      android:layout_width="160dp"
```

Comment:
Why aren't we using wrap_content here? The text might translate to something larger than we expect. Prefer to use margins to make the button longer if we want spacing around the text, or minWidth if we need it to be at least a minimum size.

Reply:
done -- ptal